pkg/api/handlers/util.go

```diff
@@ -2,6 +2,9 @@ package handlers
 
 import (
 	"fmt"
+
```

Comment:
Excuse the nitpick. But can you remove the blank line here.

I try to enforce a pattern for imports to goes like this:

```
std library imports in one block
[blank line]
external imports in one block
[blank line]
github.com/sapcc/kubernikus imports in one block
```

Unfortunately goimports `-local` flag does not fully support this and allows for addition blank lines. See https://github.com/sapcc/kubernikus/blob/master/test/gofmt.sh#L9